doc/index.rst

```diff
@@ -214,3 +214,4 @@ PennyLane is **free** and **open source**, released under the Apache License, Ve
    code/qml_variable
    code/qml_wires
    code/qml_beta
+   code/qml_qaoa
```

Comment:
Wouldn't this involve re-ordering all of the list elements?

Reply:
This would just be putting the `code/qml_qaoa` line between plugins and qchem, it's already in alphabetical order except for the beta module (which I believe is stuck at the bottom to draw less attention)

Reply:
Ah, I see